.landscaper/landscaper-service/blueprint/installation/blueprint.yaml

```diff
@@ -2,55 +2,68 @@ apiVersion: landscaper.gardener.cloud/v1alpha1
 kind: Blueprint
 
 imports:
+   # The "hostingCluster" specifies the target Kubernetes cluster into which the landscaper service will be installed.
   - name: hostingCluster
     required: true
     targetType: landscaper.gardener.cloud/kubernetes-cluster
 
+  # The "hostingClusterNamespace" is the namespace in the hosting cluster into which the landscaper service is getting installed.
+  # Only one landscaper server per namespace can be installed.
```

Comment:
these comments could be also added as schema description.
That would give us the possibility to parse the schema and generate a documentation

```yaml
  - name: hostingClusterNamespace
    required: true
    type: data
    schema:
      description: |
        The "hostingClusterNamespace" is the namespace in the hosting cluster into which the landscaper service is getting installed.
        Only one landscaper server per namespace can be installed.
      type: string
```

or we could maybe think of a dedicated field in the import definition.
```yaml
  - name: hostingClusterNamespace
    required: true
    description: |
        The "hostingClusterNamespace" is the namespace in the hosting cluster into which the landscaper service is getting installed.
        Only one landscaper server per namespace can be installed.
    type: data
    schema:
      type: string
```

Reply:
👍 
I moved it to `schema.description`.
Maybe it would be good to have something similar for targets.